Components/LandingFooter.tsx

```diff
@@ -44,16 +44,16 @@ const LandingFooter = () => {
                 <a href="https://github.com/PiyushKalyanpy/GyanaGuru" target="_blank" className={hoverEffect}><AiFillGithub className="mr-1" />Github</a>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                   <a href="https:twitter.com" className={hoverEffect}><AiFillTwitterCircle className="mr-1" />Twitter</a>
+                   <a href="https:twitter.com" className={hoverEffect}><AiFillTwitterCircle className="mr-1 transition ease duration-200 group-hover:text-sky-500" />Twitter</a>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                   <a href="https://www.linkedin.com/in/piyush-kalyan/" className={hoverEffect}><AiFillLinkedin className="mr-1" />Linkedin</a>
+                   <a href="https://www.linkedin.com/in/piyush-kalyan/" className={hoverEffect}><AiFillLinkedin className="mr-1 transition ease duration-200 group-hover:text-sky-600" />Linkedin</a>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                   <a href="https:instagram.com" className={hoverEffect}><AiFillInstagram className="mr-1" />Instagram</a>
+                   <a href="https:instagram.com" className={hoverEffect}><AiFillInstagram className="mr-1 transition ease duration-200 group-hover:text-rose-500" />Instagram</a>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                   <a href="https:facebook.com" className={hoverEffect}><AiFillFacebook className="mr-1" />Facebook</a>
+                   <a href="https:facebook.com" className={hoverEffect}><AiFillFacebook className="mr-1 transition ease duration-200 group-hover:text-blue-700" />Facebook</a>
```

Comment:
Instead of redirecting them to the other page, please use `#`, same for twitter and other link which are not correct.